onnxruntime/core/optimizer/conv_activation_fusion.cc

```diff
@@ -275,6 +291,10 @@ void RegisterConvAddReluFusionRules(SelectorActionRegistry& registry) {
   auto selector = std::make_unique<selectors::ConvAddRelu>();
   registry.RegisterSelectorAndAction(name, {{"Conv", {1, 11}}},
                                      std::move(selector), std::move(action));
+  auto selector2 = std::make_unique<selectors::ConvAddRelu>();
+  auto action2 = std::make_unique<actions::FuseConvActivationAction>(kMSInternalNHWCDomain);
```

Comment:
The ConvAddRelu Select method has this which would only make it apply to the CUDA EP wouldn't it? Is there a test that checks the new handling that uses the CUDA EP? 

```
  std::optional<NodesToOptimizeIndices> Select(const GraphViewer& graph_viewer, const Node& node) const override {
    const auto& domain = node.Domain();
    const std::string_view node_ep = node.GetExecutionProviderType();
    // only for CUDA EP
    if (node_ep != kCudaExecutionProvider) {
      return std::nullopt;
    }
```